bin/oneclick_library_import

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
 """Update the circulation manager server with new books from the
-OneClick content server."""
+OneClick content server.
+
+If you want to test with sample OneClick data on prod database, run like so:
```

Comment:
I don't think we'd ever want sample data on a prod database - do you mean a local database or a QA database?